app/helper/rtForm.php

```diff
@@ -569,6 +572,60 @@ protected function generate_textbox( $attributes ) {
 			}
 		}
 
+		/**
+		 * Generate rtmedia html inputfile in admin options.
+		 *
+		 * @access protected
+		 *
+		 * @param $attributes
+		 *
+		 * @return string
+		 * @throws rtFormInvalidArgumentsException
+		 */
+		protected function generate_inputfile( $attributes ) {
+
+			$element = 'rtFile';
+			if ( is_array( $attributes ) ) {
+
+				/* Starting the input tag */
+				$html = '<input type="file" ';
```

Comment:
check if can add html in a file template

Reply:
Will make lots of changes to existing input fields as well, we need to convert all input fields functions to html template